src/stcal/ramp_fitting/_ols_cas22/_fit.pxd

```diff
@@ -0,0 +1,32 @@
+# cython: language_level=3str
```

Comment:
Unfortunately, I was unable to eliminate this file because Cython's `enum` structures do not have a Python syntax equivalent at this time.